_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,140 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Forseti Notifier can send a variety of notifications to alert you
+of Forseti events. These notifications can be configured to be sent
+to different channels, and in different formats.
+
+### Notification Types
+
+  1. Inventory summary
+  1. Scanner summary
+  1. Violations
+  1. CSCC findings
+  
+### Notification Channels
+
+  1. Email
+  1. Slack
+  1. GCS
+
+### Notification Formats
+
+  1. csv (human-readable data)
+  1. json (structured data)
+
+## Configuring Notifier
+
+### Inventory Summary (TBD)
+
+### Scanner Summary (TBD)
+
+### Violation Notifications
+
+1. Open `forseti-security/configs/forseti_conf.yaml`.
+1. Navigate to the `notifier` > `resources` section.
+
+The following options are available, on a per resource basis. You can mix and
```

Comment:
What following options?

Reply:
Ah, they got disconnected by the example.  I'm moving them up, and the example down.  How does it look?

Reply:
Looks good over all. Instead of "mix and match", just "You can use any..." or "You can select any..." 

Rationale: mix and match is a bit of an American colloquialism that might not translate well for global users.